app/mailers/feedback_mailer.rb

```diff
@@ -7,6 +7,7 @@ def feedback(sending_user, form_values)
       to: self.class.support_email,
       subject: 'Feedback submission',
       from: from,
+      cc: from,
```

Comment:
I could be missing something here, but the `from` and `cc` are the same. On the test email sent to gatewaycommunicator, these values are different.